import { Exception } from "./exception";

export class NotFoundException extends Exception{
    constructor(user: string){
        super('${user} n√£o encontrado', 404);
    }
}